assets/js/vendor/jquery.tablesorter.js

```diff
@@ -697,11 +697,11 @@
 							extractor = ts.getParserById( ts.getData( header, configHeaders, 'extractor' ) );
 							parser = ts.getParserById( ts.getData( header, configHeaders, 'sorter' ) );
 							noParser = ts.getData( header, configHeaders, 'parser' ) === 'false';
-							// empty cells behaviour - keeping emptyToBottom for backwards compatibility
+							// empty cells behavior - keeping emptyToBottom for backwards compatibility
```

Comment:
Not going to block it because of these, but behaviour is a correct spelling... in Canada ;)

Reply:
Third-party libraries shouldn't be changed at all and reported upstream instead.

Reply:
Agreed, it didn't even click that it was in a library.  There is a new version of the library so I'll create an issue/PR to update it.